sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/event/collector/FrameworkEventCollector.java

```diff
@@ -164,4 +164,20 @@ public void collectdHotPluggingEvent(FrameworkEventDefinitions frameworkEventDef
                 frameworkEventDefinitions.getEventType(),
                 new EventInfo(frameworkEventDefinitions.getName(), description)));
     }
+
+    /**
+     * Collect OpenTelemetry Agent start event
+     *
+     * @param startMethod the method name OpenTelemetry Agent starts by
+     */
+    public void collectOtelStartEvent(String startMethod) {
```

Comment:
make this method more general and decouple it from Otel

Reply:
I think other agent start event are not sopported in Sermant. Once If a  specifc agent needs to be added into here.  We just add new interceptor and event